api/swagger.yaml

```
@@ -6302,6 +6315,75 @@ definitions:
     additionalProperties:
       type: "string"
 
+  ImageManifestSummary:
+    x-nullable: false
+    description: |
+      ImageManifestSummary represents a summary of an image manifest.
+    type: "object"
+    required: ["Id", "Descriptor", "Available", "ContentSize", "Kind"]
+    properties:
+      Id:
+        description: |
+          ID is the content-addressable ID of an image and is the same as the
+          digest of the image manifest.
+        type: "string"
+        example: "sha256:95869fbcf224d947ace8d61d0e931d49e31bb7fc67fffbbe9c3198c33aa8e93f"
+      Descriptor:
+        $ref: "#/definitions/OCIDescriptor"
+      Available:
+        description: Indicates whether all the children content is available.
```

Minor wording suggestion (and a small explanation of what "child content" means that's easier to search for):

```suggestion
        description: Indicates whether all the child content (image config, layers) is available.
```

---

Thanks!